tests/routers/test_identity.py

```
@@ -87,15 +60,17 @@ def test_import_identity_invalid_pem(self, authenticated_client):
     def test_import_identity_unexpected_error(self, authenticated_client, monkeypatch):
         """Test handling of unexpected errors during identity import."""
         from app.services.identity_service import identity_service
+
         def mock_fail(*args, **kwargs):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `mock_fail`

Add return type annotation: `NoReturn`

(ANN202)